tests/cli/workflow/fluffy/test_cli_start.py

```diff
@@ -6,16 +6,20 @@
 from cg.constants import EXIT_SUCCESS
 from cg.meta.workflow.fluffy import FluffyAnalysisAPI
 from cg.models.cg_config import CGConfig
+from cg.store.crud.read import ReadHandler
 from cg.store.models import Sample
 
 
 def test_start_available_dry(
-    cli_runner: CliRunner, fluffy_case_id_existing: str, fluffy_context: CGConfig, caplog
+    cli_runner: CliRunner, fluffy_case_id_existing: str, fluffy_context: CGConfig, caplog, mocker
 ):
     caplog.set_level("INFO")
 
     # GIVEN a case_id that does exist in database
 
+    # Temporary patch to avoid error in the test with the devel branch will be addressed in the future
```

Comment:
The mocked function will be refactored in a future pr. At that point this test should pass with the new functionality and this line will be removed

Reply:
I would add @pytest.mark.xfail(reason="flow cell on-disk check is yet to be refactored") to the test instead.

It basically marks the test as expected to fail and doesn't complain if it does 👍

Reply:
added